src/test/groovy/uk/gov/hmcts/dm/it/BaseIT.groovy

```diff
@@ -123,6 +125,16 @@ class BaseIT {
 
     }
 
+    def givenS2SRequest() {
+
+        def request = given().log().all()
+        request = request.header("serviceauthorization", serviceToken())
+                    .header("cache-control", "no-cache")
+                    .header("content-type", "multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW")
```

Comment:
What is this why do we need it???
`boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW`